x-pack/plugins/observability_solution/observability_shared/public/hooks/use_breadcrumbs.ts

```diff
@@ -58,11 +94,26 @@ export const useBreadcrumbs = (
   }>();
   const setTitle = docTitle.change;
   const appPath = getUrlForApp(app?.id ?? 'observability-overview') ?? '';
+  const chromeStyle = useObservable(getChromeStyle$());
 
-  const setBreadcrumbs = useMemo(
-    () => serverless?.setBreadcrumbs ?? chromeSetBreadcrumbs,
-    [serverless, chromeSetBreadcrumbs]
-  );
+  const setBreadcrumbs = useMemo(() => {
+    if (serverless?.setBreadcrumbs && !classicOnly)
+      return (breadcrumbs: ChromeBreadcrumb[]) =>
+        serverless?.setBreadcrumbs(breadcrumbs, { absolute });
+    if (serverless?.setBreadcrumbs && classicOnly) return noop;
```

Comment:
Q: Do we need to return `noop`- is it because of the type or can we just do an empty return?